goty-server/src/main/kotlin/com/aleinin/goty/submit/GameSubmission.kt

```diff
@@ -0,0 +1,6 @@
+package com.aleinin.goty.submit
+
+data class GameSubmission(
```

Comment:
I'm not sure how I feel about this naming convention now. This seems more like a "Game" to me than a "GameSubmission" While perhaps there is a more descriptive word than "Game" I don't think this object relates to a submission.

Reply:
I am down to change it, but my thought process is this. I don't really see it as just a "Game". A "Game" has more than just an `id` and a `title`. A "Game" is what is stored in Elasticsearch and has genres, release dates, alternate titles, etc. This represents a small subset of what a "Game" is and has a very specific use case, to represent a "Game" in a `/submission` request and response. It will never ever be used for anything else. Any other representations of a "Game" will have their own data model. For instance, a "Game" in the context of the result / winner calculation will have total # of votes and possibly tie breaker points. 

Hopefully that makes sense. I'm down with changing the name of it, but I think this represents more than just a "Game".

Reply:
I think I can follow that logic, it makes sense to me.